lib/stat_tracker.rb

```diff
@@ -1,15 +1,18 @@
 require 'csv'
+require './lib/data_module'
```

Comment:
Don't need this here anymore. Also will need to require_relative on our files moving forward

Reply:
fixed thanks for pointing it out